esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -1854,24 +1854,12 @@ void appendIOStreamContent(MemoryBuffer &mb, IFileIOStream *ios, bool forDownloa
         CWsWuFileHelper helper(nullptr);
         outIOS.setown(helper.createIOStreamWithFileName(outFile, IFOcreate));
     }
+    Owned<IStreamLineReader> lineReader = createLineReader(ios, true);
```

Comment:
Does copyFile use buffered stream?

Reply:
why? (see definitions in jfile.hpp)
You should assume it is an efficient way to copy files - if it is found that it is not, then it should be looked at, and everyone that uses it will benefit from the improvements.